### PR TITLE
chore: remove telegrafUiRefresh feature flag

### PR DESCRIPTION
**Pull request by TCL735**
Part of #3747 

- removes the feature flag `telegrafUiRefresh`
- removes tests for code that no longer exists
- drastically reduce the usage of `telegrafUiRefresh` in naming

### CODE REVIEW

**Comment by hoorayimhelping** [comment]
@TCL735 do we have tests that cover the new functionality?

File: src/dataLoaders/actions/dataLoaders.ts [hunk 1]
```diff
@@ -227,23 +226,11 @@ interface AddPluginBundle {
...
   payload: {bundle},
 })
 
-export const addTelegraf_telegrafUiRefresh = (
-  plugin: TelegrafPlugin
-): AddTelegrafTelegrafUiRefresh => ({
-  type: 'ADD_TELEGRAF_telegrafUiRefresh',
-  payload: {plugin},
-})
-
```
Comment (TCL735) [inline src/dataLoaders/actions/dataLoaders.ts:246]: This does nothing, as there was no reducer for it.

File: src/dataLoaders/actions/dataLoaders.ts [hunk 2]
```diff
@@ -227,23 +226,11 @@ interface AddPluginBundle {
   payload: {bundle: BundleName}
 }
 
-interface AddTelegrafTelegrafUiRefresh {
-  type: 'ADD_TELEGRAF_telegrafUiRefresh'
-  payload: {plugin: TelegrafPlugin}
-}
-
```
Comment (TCL735) [inline src/dataLoaders/actions/dataLoaders.ts:234]: This does nothing, as there was no reducer for it.

File: src/dataLoaders/actions/dataLoaders.ts [hunk 3]
```diff
@@ -361,11 +348,10 @@ export const addPluginBundleWithPlugins = (bundle: BundleName) => dispatch => {
   )
 }
 
-export const addTelegrafPlugin_telegrafUiRefresh = (
+export const addTelegrafPluginAsBundle = (
   plugin: TelegrafPlugin
 ) => dispatch => {
-  dispatch(addTelegraf_telegrafUiRefresh(plugin))
```
Comment (TCL735) [inline src/dataLoaders/actions/dataLoaders.ts:367]: This does nothing, as there was no reducer for it.

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**